models/ideas.js

```diff
@@ -44,6 +52,12 @@ function updateIdea(ideaId, key, value) {
 function deleteIdea(ideaId) {
     let ideas = getIdeas();
 
+    const idea = ideas.find((idea) => ideaId === idea.id);
+
+    if (!idea) {
+        throw new Error(`Error. No idea with such ID found..`);
```

Comment:
To be consistent with error messages - remove `..` at the end

Reply:
Fixed. But didn't commit.

Reply:
It's not fixed, unless it's committed :)

Reply:
COMMITED!